easybuild/easyconfigs/f/fqzcomp/fqzcomp-4.6-GCC-4.8.2.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'MakeCp'
+
+name = 'fqzcomp'
+version = '4.6'
+
+homepage = 'http://sourceforge.net/projects/fqzcomp/'
+description = """Fqzcomp is a basic fastq compressor, designed primarily for high performance.
+ Despite that it is comparable to bzip2 for compression levels."""
+
+toolchain = {'name': 'GCC', 'version': '4.8.2'}
+
+# eg. http://sourceforge.net/projects/fqzcomp/files/fqzcomp-4.6.tar.gz/download
+sources = [SOURCE_TAR_GZ]
+source_urls = [SOURCEFORGE_SOURCE]
+
+skipsteps = ['configure']
```

Comment:
@boegel: agreed, fixed this and most other bits;
negotiation pending as regards a SCALCE dep...